src/components/views/rooms/RoomTile.tsx

```diff
@@ -576,6 +597,67 @@ export default class RoomTile extends React.PureComponent<IProps, IState> {
         );
     }
 
+    private updateVoiceParticipants = (participants: IJitsiParticipant[]) => {
+        this.setState({ voiceParticipants: participants });
+    };
+
+    private renderVoiceChannel(): React.ReactElement {
+        if (!this.state.voiceParticipants.length) return null;
+
+        const faces = this.state.voiceParticipants.map(p =>
+            <BaseAvatar
+                key={p.participantId}
+                name={p.displayName ?? p.formattedDisplayName}
+                idName={p.participantId}
+                url={p.avatarURL}
```

Comment:
is this from Jitsi? If so, a comment would be great so we don't try to apply the `Media` customisation point to it.

Reply:
> Media customisation point

What is that referring to?

Reply:
a very annoyingly named class called `Media` in a `customisations` directory somewhere. It's used in forks to change how media requests are routed.